UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -54,22 +54,29 @@ public class Lungs : Organ
 	[SerializeField] private float internalBleedingCooldown = 4f;
 	private bool onCooldown = false;
 
+	private BodyPart bodyPart;
+
+	private void Awake()
+	{
+		bodyPart = GetComponent<BodyPart>();
+	}
+
 	public override void ImplantPeriodicUpdate()
 	{
 		base.ImplantPeriodicUpdate();
 		Vector3Int position = RelatedPart.HealthMaster.ObjectBehaviour.AssumedWorldPositionServer();
 		MetaDataNode node = MatrixManager.GetMetaDataAt(position);
 		var TotalModified = 1f;
-		foreach (var Modifier in RelatedPart.AppliedModifiers)
+		foreach (var Modifier in bodyPart.AppliedModifiers)
```

Comment:
since you're here, would you mind making this comply with naming convention?
```suggestion
		foreach (var modifier in bodyPart.AppliedModifiers)
```